objects/Fan-MadeAccessories.aa8b38/CleanUpHelper.26cf4b.ttslua

```diff
@@ -259,42 +271,50 @@ function getTrauma()
   end
 end
 
--- get rid of bless/curse tokens via bless / curse manager
+-- get rid of bless/curse tokens via bless/curse manager
 function removeBlessCurse()
   local BlessCurseManager = getObjectFromGUID("5933fb")
+
   if BlessCurseManager ~= nil then
     BlessCurseManager.call("doRemove", "White")
+  else
+    printToAll("Bless / Curse manager could not be found and thus bless/curse tokens were skipped.", "Yellow")
   end
 end
 
 -- remove drawn lines
 function removeLines()
-  Global.setVectorLines({})
+  if options["removeDrawnLines"] then
+    printToAll("Removing vector lines...", "White")
+    Global.setVectorLines({})
+  end
 end
 
 -- discard all hand objects
 function discardHands()
```

Comment:
Seems like this should be gated as part of cleaning up the playermat?  If I don't want my cards cleaned off the mat it doesn't seem likely I want their hand trashed.